resources/config.edn

```
@@ -5,4 +5,5 @@
       :subname     "//localhost:5432/caseworker"}
  :environment "development"
  :google-client-id "174928214934-m0rbjmumbsa0p5ubtuq7ljpe60evoen9.apps.googleusercontent.com"
```

Same here. This is not as dangerous to leave laying around, but if you are already going the secrets management route, you should put this here as well.

---

same with this one: `src/caseworker/config.clj` overrides these all values like this with ones provided via environment variables. i'm very keen though to make sure that if people don't provide their own, we don't just silently use the default, but rather blow up noticeably. i think using the `:env` key in the `project.clj` hopefully helps towards this end.